src/api/callback/controller.js

```diff
@@ -0,0 +1,15 @@
+const callbackController = {
+  options: {
+    payload: {
```

Comment:
I think these are all the defaults, so no need to add them - double check the docs